TASK№2/TASK№2/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TASK_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Диаметр внешнего кольца: ");
+            double D = Convert.ToDouble(Console.ReadLine());
```

Comment:
Класс `Convert` считается устаревшим. Не стоит обращаться к нему без особой необходимости. 
Локальные переменные принято называть с маленькой буквы, а имена лучше придумывать значимые.
```suggestion
            double exteriorD = double.Parse(Console.ReadLine());
```